tests/test_config.py

```diff
@@ -613,15 +699,46 @@ def test_tag_filter():
     assert "probes.lmrc.SexualContent" in found
 
 
+# when provided an absolute path as `reporting.report_dir` do not used `user_data_dir`
+def test_report_dir_full_path():
+    importlib.reload(_config)
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+
+        report_path = Path(tmpdir).absolute()
+        with tempfile.NamedTemporaryFile(buffering=0, delete=False) as tmp:
+            tmp.write(
+                "\n".join(
+                    [
+                        f"---",
+                        f"reporting:",
+                        f"  report_dir: {report_path}",
+                    ]
+                ).encode("utf-8")
+            )
+            tmp.close()
+            garak.cli.main(
+                f"-m test.Blank --report_prefix abs_path_test -p test.Blank -d always.Fail --config {tmp.name}".split()
+            )
+            os.remove(tmp.name)
```

Comment:
context handler doesn't remove it?

Reply:
No the option `delete=False` leaves it in place, in theory that option could be changed to allow the context handler to address it. The difference in `windows` behavior though has lead me to be more explicit with temp files in the the projects tests.

Reply:
ah yeah, i've had similar experiences. i guess we should be grateful that `.close()` is sufficient to not require an additional `.flush()`.